frontend/src/views/main/project/SuiteTestExecutionList.vue

```diff
@@ -136,5 +140,54 @@ async function removeTest(suiteTest: SuiteTestDTO) {
     }
   });
 }
+
+async function debugTest(result: SuiteTestExecutionDTO, suiteTest: SuiteTestDTO) {
+  let inputs = [];
+  if (props.execution.inputs.length > 0) {
+    inputs = props.execution.inputs;
+  } else {
+    // This is a bit hacky, but there is no other way because the test suite input list is never guaranteed to be present :(
+    inputs.push({
+      name: "model",
+      type: "BaseModel",
+      isAlias: false,
+      value: result.inputs["model"],
```

Comment:
we should actually be looking at argument type, not name. If I upload a test whose input is called "my_model" this won't work